### PR TITLE
fix: add create cursor util

### PR DESCRIPTION
**Pull request by magrinj**
Followup of #9053

### CODE REVIEW

File: packages/twenty-front/src/modules/apollo/utils/createCursor.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  }
+
+  const payload: {
+    id: string;
+    position?: number;
+  } = {
+    id: record.id,
+  };
+
+  if ('position' in record) {
```
[the remark this answers is not in the document]
Comment (magrinj) [reply packages/twenty-front/src/modules/apollo/utils/createCursor.ts:15]: Hmm, I don't think so, it should works in both cases, when it's defined and when it's not.
If we have position this one is added to the payload, if not we're just using the payload containing the id

[the remark this answers is not in the document]
Comment (Weiko) [reply packages/twenty-front/src/modules/apollo/utils/createCursor.ts:16]: I think that's what my suggestion does. It should work in both case and if position is undefined, the position in the payload will be undefined?

File: packages/twenty-front/src/modules/apollo/utils/encodeCursor.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import { ObjectRecord } from '@/object-record/types/ObjectRecord';
+import { isDefined } from '~/utils/isDefined';
+
+export const encodeCursor = (record: ObjectRecord) => {
+  if (!('id' in record) || !isDefined(record.id)) {
+    throw new Error('Record does not have an id');
+  }
+
+  const payload: {
```
Comment (Weiko) [inline packages/twenty-front/src/modules/apollo/utils/encodeCursor.ts:9]: Not sure if we need to instantiate this now that you don't modify it?

**Review by Weiko — APPROVED** [review]

File: packages/twenty-front/src/modules/apollo/utils/encodeCursor.ts (hunk 2 above)
Comment (magrinj) [reply packages/twenty-front/src/modules/apollo/utils/encodeCursor.ts:9]: I don't want to put all records values inside the payload, it's why I'm instantiating a new object

**Pull request merged**